mm/bootmem.c

```diff
@@ -75,7 +75,7 @@ unsigned long __init bootmem_bootmap_pages(unsigned long pages)
 /*
  * link bdata in order
  */
-static void __init link_bootmem(bootmem_data_t *bdata)
+void __init link_bootmem(bootmem_data_t *bdata)
```

Comment:
does someone use link_bootmem in an external file ? if not, keep it as the original static function.